### PR TITLE
Updates 2023-01-28

### PR DESCRIPTION
**Pull request by lsocrate**
Updating a lot of cards. Still missing a few.

### CODE REVIEW

File: server/game/cards/19.1-AS01/Dragon/VillageDoshin.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                        inPlay &&
+                        byAbility &&
+                        fromOpponent
+                    );
+                }
+            },
+
+            gameAction: AbilityDsl.actions.conditional({
+                condition: (context) =>
+                    context.player.opponent.fate >= this.doshinTax(context),
```
Comment (marcsoiferman) [inline server/game/cards/19.1-AS01/Dragon/VillageDoshin.js]: You need to also check if your opponent here can spend fate.
`AbilityDsl.actions.loseFate().canAffect(context.player.opponent, context)`

File: server/game/cards/19.1-AS01/Neutral/MantisRaider.js [hunk 2]
```diff
@@ -18,13 +18,13 @@ class MantisRaider extends DrawCard {
         });
 
         this.action({
-            title: 'Give this character +2/+2',
+            title: 'Give this character +1/+0',
```
Comment (marcsoiferman) [inline server/game/cards/19.1-AS01/Neutral/MantisRaider.js]: If you're in here editing, you don't need to say it gets +0 pol.

Comment (lsocrate) [reply server/game/cards/19.1-AS01/Neutral/MantisRaider.js]: Fixed

File: server/game/cards/19.1-AS01/Dragon/VillageDoshin.js (hunk 1 above)
Comment (lsocrate) [reply server/game/cards/19.1-AS01/Dragon/VillageDoshin.js]: Fixed

**Review by marcsoiferman — APPROVED** [review]

**Pull request merged**